core/src/main/java/io/stargate/core/metrics/impl/MetricsImpl.java

```diff
@@ -1,14 +1,36 @@
 package io.stargate.core.metrics.impl;
 
 import com.codahale.metrics.MetricRegistry;
+import io.micrometer.core.instrument.Clock;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.prometheus.PrometheusConfig;
+import io.micrometer.prometheus.PrometheusMeterRegistry;
+import io.prometheus.client.CollectorRegistry;
+import io.prometheus.client.dropwizard.DropwizardExports;
 import io.stargate.core.metrics.api.Metrics;
+import io.stargate.core.metrics.api.MetricsScraper;
 
-public class MetricsImpl implements Metrics {
+public class MetricsImpl implements Metrics, MetricsScraper {
 
   private final MetricRegistry registry;
 
+  private final PrometheusMeterRegistry prometheusMeterRegistry;
+
   public MetricsImpl() {
     registry = new MetricRegistry();
+    prometheusMeterRegistry = initPrometheusMeterRegistry(registry);
+  }
+
+  private PrometheusMeterRegistry initPrometheusMeterRegistry(MetricRegistry metricRegistry) {
+    // note that we are adding the dropwizard exports to the CollectorRegistry that we will pass to
+    // the meter registry
+    DropwizardExports dropwizardExports = new DropwizardExports(metricRegistry);
+    CollectorRegistry collectorRegistry = new CollectorRegistry();
+    collectorRegistry.register(dropwizardExports);
+
+    PrometheusMeterRegistry meterRegistry =
+        new PrometheusMeterRegistry(PrometheusConfig.DEFAULT, collectorRegistry, Clock.SYSTEM);
```

Comment:
Is it safe to rely on: `PrometheusConfig.DEFAULT`? What's does it provide?

Reply:
Default config is no config (empty).